### PR TITLE
Exercise26

### PR DESCRIPTION
**Pull request by clucksupremacy**

### CODE REVIEW

File: exercise-26.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+function rent_prices() {
+    let average_rent = 0;
+    let apts_registered = [];
+    let number_input = 0;
+    let sum = 0;
+
+    function collect_apt_prices() {
+        const prompt = require("prompt-sync")({ sigint: true });
```
Comment (Aceeri) [inline exercise-26.js]: Try moving this outside of the functions, people tend to use these are global imports.

File: exercise-26.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        } else if (isNaN(input)) {
+            collect_apt_prices();
+        } else {
+            number_input = Number(input);
+            apts_registered.push(number_input);
+            collect_apt_prices();
+        }
+    } collect_apt_prices();
+
+    function calculate_avg_price() {
```
Comment (Aceeri) [inline exercise-26.js]: Also move these functions outside of the function above.

File: exercise-26.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        average_rent = sum / (apts_registered.length);
+        // average_rent = apts_registered.reduce((a, b) => a + b, 0); 
+            // the start of an alternative method
+    
+        console.log(apts_registered.length, "apartments have registed. The average price for rent is ",     average_rent, "$"); // keeping the funky spelling errors
+
+        return average_rent;
+    } calculate_avg_price();
+
+    function compare_prices() {
```
Comment (Aceeri) [inline exercise-26.js]: Actually try just doing these with loops instead of recursive functions.

**Review by Aceeri — CHANGES_REQUESTED** [review]

**Comment by clucksupremacy** [comment]
Changes made @Aceeri

File: exercise-26.js [hunk 4]
```diff
@@ -0,0 +1,58 @@
+let average_rent = 0;
+    let apts_registered = [];
+    let number_input = 0;
+    let sum = 0;
```
Comment (Aceeri) [inline exercise-26.js]: Try moving these into the functions they are related to using parameters and returning

**Pull request merged**